libraries/AP_Proximity/AP_Proximity_Backend.h

```diff
@@ -42,6 +42,16 @@ class AP_Proximity_Backend
     // handle mavlink messages
     virtual void handle_msg(const mavlink_message_t &msg) {}
 
+#if AP_SCRIPTING_ENABLED
+    // handle Lua obstacle messages
+    virtual bool set_distance_min_max(float min, float max) { return false; }
+    virtual bool handle_script_distance_msg(float dist_m, float azimuth_deg, float elevation_deg, bool push_to_boundary) { return false; }
```

Comment:
I think we should probably rename "azimuth" to "yaw" or "heading".  "azimuth" is a fine word on it's own but we should try and avoid synonyms where possible.